### PR TITLE
fix: packages listed twice in watch mode

### PR DESCRIPTION
**Pull request by maciej-ka**
### Description
Fixes: https://github.com/vercel/turborepo/issues/9538

The `WatchClient::new` already lists packages as a part of `start_ui`.
Therefore, an additional prelude print in `WatchClient::start` appears redundant.

### Testing Instructions
1. setup a [devturbo alias](https://github.com/vercel/turborepo/blob/main/CONTRIBUTING.md#manually-testing-turbo)
1. clone https://github.com/remotion-dev/remotion
1. run `devturbo watch make`

### CODE REVIEW

**Comment by ijjk** [comment]
Allow CI Workflow Run

- [ ] approve CI run for commit: c94172ddcd29a51c8da9ac0c67c508f393549f02

Note: this should only be enabled once the PR is ready to go and can only be enabled by a maintainer

**Comment by chris-olszewski** [comment]
@maciej-ka This PR is great! Our CI requires me to approve a run for each additional commit so please do not merge main unless necessary.

**Review by chris-olszewski — APPROVED** [review]
Dziękuję!

**Comment by chris-olszewski** [comment]
Something that's a little more advanced, but self contained is https://github.com/vercel/turborepo/issues/8476. 

The issue is that we don't remove and restore directories in the output. For the most case this doesn't matter, but if something created a symlink in the output it results in unexpected behavior upon a cache hit. I believe the fix will be adding some logic in [this method](https://github.com/vercel/turborepo/blob/main/crates/turborepo-cache/src/cache_archive/restore_directory.rs#L67).

I added a test that displays this behavior in https://github.com/vercel/turborepo/pull/9646

If you have JS experience there are many [issues related to our examples](https://github.com/vercel/turborepo/issues?q=is%3Aopen+is%3Aissue+label%3A%22area%3A+examples%22) that could use a pair of eyes. [This one](https://github.com/vercel/turborepo/issues/7747) is fixing some peer dependencies

**Comment by maciej-ka** [comment]
Thanks!
Both the cache and example problems looks like a great match 🥬🐇

**Pull request merged**
